src/components/RenderReview.jsx

```diff
@@ -2,6 +2,7 @@ import { Link } from "react-router-dom"
 
 function RenderReview({review}) {
 
+
```

Comment:
Lines 16-20 is this actually a list? Could some more suitable html tags be used here to present the data?